tools/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
 dependencies:
   args: ^2.1.0
   file: ^6.1.0
-  flutter_plugin_tools: ^0.7.3
+  flutter_plugin_tools: ^0.8.2
```

Comment:
Bumping up minor version when major version is non-positive may introduce breaking changes. Also this change is not relevant to `flutter_blue_tizen`. Please revert it.

Reply:
I advised him to try the latest one because the new version allowed use of local `.clang-format` files but the old version didn't. (He should've mentioned about this in the PR comment.)

Reply:
Oh okay, the tool logic doesn't seem to break at the moment. I'll see if there are any required changes in the tool code.